.env.example

```diff
@@ -45,3 +45,4 @@ NEXT_PUBLIC_REDEFINE_API=
 NEXT_PUBLIC_SOCIAL_WALLET_OPTIONS_STAGING=
 NEXT_PUBLIC_SOCIAL_WALLET_OPTIONS_PRODUCTION=
 
+NEXT_PUBLIC_WEB3MODAL_PROJECT_ID=
```

Comment:
Looks like this would be a required env var. We should mention it in the readme.